pkg/operation/botanist/component/kubeapiserver/networkpolicies.go

```diff
@@ -134,7 +140,18 @@ func (k *kubeAPIServer) reconcileNetworkPolicyAllowKubeAPIServer(ctx context.Con
 					Protocol: &protocol,
 					Port:     &portEtcd,
 				}},
-			}},
+			},
+				{
+					To: []networkingv1.NetworkPolicyPeer{{
+						PodSelector: &metav1.LabelSelector{
+							MatchLabels: vpnseedserver.GetLabels(),
+						},
+					}},
+					Ports: []networkingv1.NetworkPolicyPort{{
+						Protocol: &protocol,
+						Port:     port,
+					}},
+				}},
```

Comment:
Please fix the indentation with regards to the block above. It is probably best to add a line break between the two opening brackets after `NetworkPolicyEgressRule`.